### PR TITLE
Add script for random traffic data generation

### LINKED ISSUE #1303: Add trigger data for testing
@jbarrett I was talking with @alohaas about this a bit - would it be possible to have some test data for triggering so we can interact with the graphs without breaching data we aren't supposed to have access to?

Could we either a) have a means of authing so we can get real data for IAs we can access, or b) have some randomized sets of data that show sensible triggering (i.e, high triggering for some IAs, medium for most, low for a few).

Currently I only get the `Not enough data` message.

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
`script/ddgc_generate_traffic.pl` generates random traffic data for local testing.

Fixes #1303 

screenshot-2016-05-16-22 05 04_784_466

#### TO-DO

- [x] More realistic data (not 3 million queries!)
- [x] Truncate tables on update (@jdorweiler)

/cc @jdorweiler @MariagraziaAlastra @jbarrett

### CODE REVIEW

File: script/ddgc_generate_traffic.pl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+my $days = \@days;
+
+sub generate_bounded_counts {
+    my ($min, $max) = @_;
+    map { int(rand($max+$min)) + $min } (1..31);
+}
+
+my %bounds = (
+    small => [0, 3_000],
+    large => [1_000, 100_000],
```
Comment (GuiltyDolphin) [inline script/ddgc_generate_traffic.pl]: I think we can probably just have the max bound. The min bound becomes less important with large values.

File: script/ddgc_generate_traffic.pl [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+my @ids = $d->rs('InstantAnswer')
+    ->search({dev_milestone => 'live'})
+    ->get_column('meta_id')
+    ->all;
+
+my $today = DateTime->now();
+my $last_month = $today->clone()->subtract(days => 31);
+
+my @days = map { $last_month->clone->add(days => $_)->strftime('%F') } (1..31);
```
Comment (GuiltyDolphin) [inline script/ddgc_generate_traffic.pl]: Might be worth pulling the number of days out so we can specify how many days of data we want to generate.

File: script/ddgc_generate_traffic.pl [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+
+$ENV{DDGC_IA_AUTOUPDATES} = 0;
```
Comment (GuiltyDolphin) [inline script/ddgc_generate_traffic.pl]: @jdorweiler Do I need this? I yanked this from `ddgc_populate_ia_dev`.

**Comment by GuiltyDolphin** [comment]
@jdorweiler @MariagraziaAlastra Okay - I think I'm done? You can use the `--help` and `--man` options to see the command options.

If you would prefer more realistic data rather we'll probably need to employ the help of a statistics package.

**Comment by GuiltyDolphin** [comment]
@jdorweiler Jeez... The build for this thing takes *ageees*; shouldn't this PR be using the master cache?

File: script/ddgc_generate_traffic.pl [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+use FindBin;
+use lib $FindBin::Dir . "/../lib";
+
+use DDGC;
+use List::MoreUtils qw(zip);
+use List::Util qw(any pairs);
+use DateTime;
+
+use Moo;
+use MooX::Options
```
Comment (jdorweiler) [inline script/ddgc_generate_traffic.pl]: :+1:

File: script/ddgc_generate_traffic.pl (hunk 3 above)
Comment (jdorweiler) [inline script/ddgc_generate_traffic.pl]: You can leave it.

**Comment by jdorweiler** [comment]
LGTM, thanks for making this :+1:

**Pull request merged**